### DISCUSSION TITLE
Sodium license change and Iris' stance of it

### DISCUSSION THREAD

**Discussion by NoaHimesaka1873** [IrisShaders/Iris#2413] [General]
Sodium changed a license to Polyform Shield which has noncompete clause, making it not free nor open-source. Will Iris continue to support Sodium after this? IMO it should support Embeddium which is free and open source fork instead of now nonfree Sodium now it also supports Fabric.

**Comment by IAmREGE** [comment]
> Sodium changed a license to Polyform Shield which has noncompete clause, making it not free nor open-source. Will Iris continue to support Sodium after this? IMO it should support Embeddium which is free and open source fork instead of now nonfree Sodium now it also supports Fabric.

Re: Sodium changed the license to Polyform Shield which has a noncompete clause, making it not free nor open-source. Will Iris continue supporting Sodium after this? IMO it should turn to support Embeddium which is a free and open source fork, instead of the present nonfree Sodium. Now Embeddium also supports Fabric.

**Comment by IAmREGE** [comment]
I agree that if Iris still wants to keep its LGPL license :)
